spec/services/proofing/lexis_nexis/ddp/response_redacter_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe Proofing::LexisNexis::Ddp::ResponseRedacter do
+
+  let(:json) { 
+    Proofing::LexisNexis::Ddp::ResponseRedacter.
+      redact(sample_hash)
+  }
+
+  describe 'self.redact' do
+    context 'hash with unknown keys' do
+      let(:sample_hash) {
+        {
+          'unknown_key' => 'dangerous data',
+          'first_name' => 'unsafe first name',
+          'ssn_hash' => 'unsafe ssn hash',
+        }
+      }
+      it 'redacts unknown keys' do
+        expect(json.values).to eq(['[redacted]']*3)
+      end
+      it 'keeps redacted keys' do
+        expect(json.keys.length).to eq(3)
+      end
+    end
+
+    context 'preserves known keys' do
+      let(:sample_hash) {
+        {
+          'review_status' => 'safe value',
+          'summary_risk_score' => 'safe value',
+          'fraudpoint.score' => 'safe value'
+        }
+      }
+      it 'redacts unknown keys' do
+        expect(json.values).to eq(['safe value']*3)
+      end
```

Comment:
same, I think the whole body would be easier to see
```suggestion
      it 'allows known keys' do
        expect(json).to eq(
          'review_status' => 'safe value',
          'summary_risk_score' => 'safe value',
          'fraudpoint.score' => 'safe value'
        )
      end
```

and maybe even combining this spec with the above? One body that has both known + unknown keys?